### ISSUE TITLE
Discovery: New approval process for the release

### ISSUE THREAD

**Issue by timwright12** [department-of-veterans-affairs/va-mobile-app#6814]
The current release approval process is for someone to comment /approve v2.12.0 on the release ticket. Having someone add free form text to approve the release is ripe for issues. We've already seen some typos causing the release to not be approved. We need something more bulletproof (probably a button only available to approved users/groups)

- [ ] Let's gather a list of options and research how this is handled at other places.

**Comment by BeccaTupaj** [comment]
@jbergman-oddball and @timwright12 I have added this to our Project and to the current sprint. Right now this is a floating ticket, I dont know yet what epic this could be tied to but want to make sure we are tracking it!

**Comment by jbergman-oddball** [comment]
I have found a way to complete this by using the environments option and using this for the approval.  We can add this into the current workflow right after the issue is created. It will then send out a notification (via email) to the approvers with a link to the review.  The approvers can approve and the rest of the workflow will run.  There will also be a link to the approval screen in the issue itself. 

![Image](https://github.com/user-attachments/assets/5b92c170-a0ce-4331-9ebf-9230d0485639)

Clicking on the link will bring the user to this screen

![Image](https://github.com/user-attachments/assets/20439db2-4787-4bf0-8bf7-ffb3402036ed)

If they have access to approve, they wi;; see `review deployments`
The user can then check the env option, add a note and click approve.

![Image](https://github.com/user-attachments/assets/faa7b4aa-1620-4d2d-a752-28c5025ec9ef)

the rest of the job will run and complete.

![Image](https://github.com/user-attachments/assets/45efa3e2-4bcf-48c9-8ab4-1a2a71f37959)

![Image](https://github.com/user-attachments/assets/a25b8576-0ca1-4ff2-8786-3ea6fbeaab6e)


We can add this in place of the `/approve` command that is run when the release branch is cut.  We will set the expiration to 14 days for approval.  After the approval is granted, the rest of the actions will kick off, slack channel, closing of the issue, etc.

@BeccaTupaj  What are next steps here?  I believe this would be a good task for the demo.

**Comment by jbergman-oddball** [comment]
Also worth noting that the approvers will get an email as well. the button will take them directly to the run where they can approve as well.

![Image](https://github.com/user-attachments/assets/7feab93f-dddd-491f-a8f6-8d5598994d32)

**Comment by timwright12** [comment]
@BeccaTupaj I like this, I think we should show it off at sprint demo and see if Ryan would like us to implement this version.

@jbergman-oddball we should probably come with a solid level of effort estimate

**Comment by BeccaTupaj** [comment]
@timwright12 and @jbergman-oddball Has this ticket been completed?

**Comment by jbergman-oddball** [comment]
@BeccaTupaj, yes I believe the discovery work is complete.  We need to create a ticket for the implementation
